ui/component/comment/view.jsx

```diff
@@ -79,6 +214,12 @@ function Comment(props: Props) {
             </div>
           )}
         </div>
+        <div>
+          {isReplying ? (
+            <CommentReply uri={uri} parentId={commentId} setReplying={setReplying} />
```

Comment:
`isReplying && <Comment ... />` 😛 

Recursive component!

Reply:
CommentReply is the component that comes up to type in your reply, not an actual Comment.

Reply:
Ah right. In that case, it should use `<CommentCreate />` since it's almost exactly the same.

Reply:
The main difference is this cancel button
![image](https://user-images.githubusercontent.com/6563959/73681183-bb159480-46b5-11ea-9e11-6e936ec95c43.png)

I would use CommentCreate if I can get that cancel button somehow.

Making a modification to CommentCreate to include Cancel button is a possibility but might be weird since we won't want there to be one for the top level comments.
![image](https://user-images.githubusercontent.com/6563959/73681557-7dfdd200-46b6-11ea-9753-3c4a619d571e.png)

Reply:
Yep. It should only include the cancel button if it's a reply.

Reply:
@seanyesmunt So should I keep the CommentReply component?